### ISSUE TITLE
cant reproduce results on images having sky/lighter back ground

### ISSUE THREAD

**Issue by farazBhatti** [MathiasGruber/PConv-Keras#40]
@MathiasGruber Hi, I've used your code to get pretty decent results but when ever there is a lighter background in image such as sky , some noise is added as a result. But your deployed model on  [ www.fixmyphoto.ai]( www.fixmyphoto.ai) works fine. is there something missing in this code base ?

![Screenshot from 2019-07-24 10-25-26](https://user-images.githubusercontent.com/37732181/61767829-e9f5a880-adfe-11e9-99a3-31b5653ed37f.png)
for reference..

**Comment by ghost** [comment]
**My private opinion** 
Would you check your model load pytorch_vgg16.h5 at each phase?
This is important because pre-trained vgg16 net is required for perceptual loss and style loss.
It not, it might give you blurriness or unexpected results.

Best regards,

**Comment by nywang2019** [comment]
@farazBhatti  see there are some white blocks at the upper edge of your result, and a black block at the upper right corner. did you resolve this problem? my result images also have a white area at the upper left corner,and a black rect block at the upper right corner. why??? anyone can tell me? @Ferruginous-Hawk0215
